comms/discovery/rpcz/chat.go

```diff
@@ -46,8 +63,24 @@ func chatCreate(tx *sqlx.Tx, userId int32, ts time.Time, params schema.ChatCreat
 		if err != nil {
 			return err
 		}
+
+	}
+
+	for _, blast := range blasts {
+		_, err = tx.Exec(`
+		insert into chat_message
+			(message_id, chat_id, user_id, created_at, blast_id)
+		values
+			($1, $2, $3, $4, $5)
+		on conflict do nothing
+		`, params.ChatID+blast.BlastID, params.ChatID, blast.FromUserID, ts, blast.BlastID)
```

Comment:
I'd argue that the timestamp should come from the blast to ensure order for multiple blasts